src/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixer.php

```diff
@@ -128,7 +129,6 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                 T_PROTECTED,
                 T_PUBLIC,
                 T_STATIC,
-                T_VAR,
```

Comment:
This list is for tokens before the function (see line 136), so `T_VAR` here is both incorrect and misleading.